contracts/cw20-ics20/src/contract.rs

```diff
@@ -251,9 +298,10 @@ pub fn query_channel(deps: Deps, id: String) -> StdResult<ChannelResponse> {
 
 fn query_config(deps: Deps) -> StdResult<ConfigResponse> {
     let cfg = CONFIG.load(deps.storage)?;
+    let admin = ADMIN.get(deps)?.unwrap_or_else(|| Addr::unchecked(""));
```

Comment:
Nice. Wouldn't `"None"` be better here? I think we used that in the past.

Reply:
In this contract, we don't allow None, so it doesn't map much to the ADMIN, I kind of force it